lighthouse-core/test/lib/page-functions-test.js

```diff
@@ -85,32 +85,43 @@ describe('Page Functions', () => {
 
   describe('get outer HTML snippets', () => {
     it('gets full HTML snippet', () => {
-      assert.equal(pageFunctions.getOuterHTMLSnippet(
-        dom.createElement('div', '', {id: '1', style: 'style'})), '<div id="1" style="style">');
+      const elem = dom.createElement('div');
```

Comment:
wow, `dom.js` in here. I even thought it looked "pretty handy" to have in here in the original review :)

I feel like it shouldn't be crossing out of the report anymore. Since it's basically equivalent to `document.createElement()` calls now, do you mind cutting out `dom.js` now?